beacon_node/eth2-libp2p/src/rpc/handler.rs

```diff
@@ -507,9 +502,9 @@ where
                         unreachable!("Coding error: Poisoned substream");
                     }
                 }
-            }
-            // We do not send errors as responses
-            RPCEvent::Error(..) => {}
+            } // TODO: don't we?
```

Comment:
The `RPC:Error` type is used for reporting errors that occur in the protocol. Like stream timeouts, and protocol negotiation failures etc. These don't get sent outwards. 
There is an `RPCEncoded::Error` type which gets sent. Originally was an `RPCErrorResponse` which had `Success` and `Error` variants.